src/Timeline.cpp

```diff
@@ -1021,9 +1021,6 @@ std::vector<Clip*> Timeline::find_intersecting_clips(int64_t requested_frame, in
 	float min_requested_frame = requested_frame;
 	float max_requested_frame = requested_frame + (number_of_frames - 1);
 
-	// Re-Sort Clips (since they likely changed)
```

Comment:
This was being called inside the `Timeline::GetFrame()` method, and it is simply not necessary to sort the list of clips here, since clips change much less frequently. Instead sort clips on these methods: (i.e. AddClip or ApplyJsonDiff or SetJson methods).